README.md

```diff
@@ -54,6 +105,15 @@ curl -v http://localhost:8080/api/v1/metadata
 - [Architecture](./docs/ARCHITECTURE.md): architectural decisions behind designing harbor-scanner-trivy.
 - [Releases](./docs/RELEASES.md): how to release a new version of harbor-scanner-trivy.
 
+## Contributing
+
+Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull
```

Comment:
We do have a standard COC (see https://github.com/aquasecurity/.github/blob/master/CODE_OF_CONDUCT.md - the .github files get applied to all projects by default)

Reply:
Update the link.